.github/workflows/ci.yml

```diff
@@ -10,17 +10,40 @@ on:
 
 jobs:
   scalafmt: 
+    name: Formatting
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2.3.4
-        with:
-          submodules: true
-          fetch-depth: 0
       - uses: olafurpg/setup-scala@v11
         with:
           java-version: graalvm-ce-java11@20.1.0
       - name: Check formatting
         run: ./bin/scalafmt --test
+  integrations: 
+    name: Build integrations tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+        jdk: [graalvm-ce-java11@20.1.0]
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          submodules: true
```

Comment:
I know in the past we hit on issues with not include `fetch-depth: 0` in the checkout. If I'm not mistaken we even it on it in Bloop in the past. I'd probably include that here.

Reply:
Do you remember the kind of issue?

As long as it does not happen again, I would prefer not to include the `fetch-depth: 0` here.

Reply:
The issue was connected to releasing, we would get a wrong dynamic version, but it's not an issue in tests themselves. We only use `fetch-depth: 0` in the release job of metals.

Reply:
> but it's not an issue in tests themselves.

Ah yea true, good point.